common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -49,8 +49,10 @@ def build_partition_values(all_user_partitions, selected_user_partition):
         # Add "No selection" value if there is not a valid selected user partition.
         if not selected_user_partition:
             SplitTestFields.user_partition_values.append(SplitTestFields.no_partition_selected)
-        for user_partition in all_user_partitions:
-            SplitTestFields.user_partition_values.append({"display_name": user_partition.name, "value": user_partition.id})
+        for user_partition in get_split_user_partitions(all_user_partitions):
```

Comment:
Note to reviewers: the changes in this file and the associated test file are also in #6397.